splinepy/helpme/create.py

```diff
@@ -298,6 +298,306 @@ def revolved(
     return type(spline)(**spline_dict)
 
 
+def swept(
+    cross_section,
+    trajectory,
+    cross_section_normal=None,
+    auto_refinement=False,
```

Comment:
As mentioned at the beginning, this should be optional insertion points. as we saw offline, this didn't yield a good result

Reply:
`auto_refinement` is deleted now